src/index.html

```diff
@@ -1,35 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
 </head>
+
 <body>
+  <main id="lb-main">
     <h1>Leaderboard</h1>
-    <div class="board">
-        <div class="rec-scores">
-            <div class="scores1">
-                <div class="top1">
-                    <h2>Recent Scores</h2>
-                    <button class="refreshed">Refresh</button>
-                </div>
-                <div class="bottom1">
-                    <ul class="scorelist">
-                        
-                    </ul>
 
-                </div>
-            </div>
-        </div>
-        <div class="addscores1">
-            <div class="adding">
-                <h2>Add your score</h2><br><br>
-                <input class="name" type="text" placeholder="name"><br><br>
-                <input class="score" type="text" placeholder="score"><br><br>
-                <button class="add">Submit</button>
-            </div>
+    <div class="structure">
+      <section class="scores-section">
+        <div class="recent-scores">
+          <h2>Recentscores</h2>
+          <button type="submit" id="Refresh1">Refresh</button>
```

Comment:
- [ ] According to the project requirement, the `Refresh button` is meant to make an `API` request and get all the current scores

> Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).

- But currently, clicking the button doesn't trigger any event, Please look into this. :smile: :wave: